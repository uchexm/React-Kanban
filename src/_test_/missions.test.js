import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Missions from '../components/missions/missions';
import store from '../redux/configureStore';

describe('Test for Mission', () => {
  test('should render', () => {
    const msn = {
      id: 1,
      name: 'dfg',
      description: 'ghdj',
      reserved: false,
    };
    const tree = renderer.create(
      <Provider store={store}>
        <Missions mission={msn} key={1} />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
