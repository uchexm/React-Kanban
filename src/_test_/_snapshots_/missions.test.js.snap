// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Mission should render 1`] = `
<div>
  <table>
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th
          style={
            Object {
              "paddingLeft": "30px",
            }
          }
        >
          Status
        </th>
        <th>
            
        </th>
      </tr>
    </thead>
  </table>
  <footer>
       
  </footer>
</div>
`;